Docs: Add install doc pages (#186)
kubernetes-sigs/kueue: docs/README.md

# Kueue documentation

> Welcome to Kueue - Kubernetes native APIs and controller for queueing jobs.

Reviewer: ```suggestion
Welcome to Kueue!
Kueue is a Kubernetes native set of APIs and controllers for queueing jobs.
```

Not sure if the plural is accurate :)

Reviewer 2: actually, it is accurate :)

> Welcome to Kueue - A Kubernetes native APIs and controller for queueing jobs.

Reviewer 2: ```suggestion
Welcome to Kueue - Kubernetes native APIs and controller for queueing jobs.
```

## Understand Kueue

Learn about Kueue and its fundamental concepts.

[**View concepts**](concepts)

> ## Quick start

Reviewer 3: why do we call it quick start? is there a slow start :)

perhaps name it Setup (just as the reference header the section is pointing to)

Author: jejeje good one :P 

lot's of documentation pages have a small "quick start" section. but I agree with you here, will address in a min

> For a step-by-step instructions on how to get Kueue running on your cluster.

Reviewer 2: ```suggestion
Follow step-by-step instructions on how to get Kueue running on your cluster.
```

[**View Setup**](setup)

## Learn how to use Kueue

> Look up common tasks and how to perform them using a short sequence

Reviewer: ```suggestion
Find common Kueue configuration tasks and learn how to perform them.
```

Reviewer 2: It's not just configuration. Batch admins do configuration, but Batch users run and monitor their jobs.

Reviewer: Gotcha. Maybe:
```
Learn what are and how to perform the available Kueue tasks, based on the user type.
```

Author: this was not part of the pr :)

# Kueue documentation

> Welcome to Kueue - A Kubernetes native way for queueing workloads

Reviewer 2: way?

Perhaps system? Or you can be more explicit saying "APIs and controller"

Reviewer 2: And using the word "job" will be more familiar for most people before they read "concepts"

> ## Quick start

Reviewer 2: Let's put this section after `Understand Kueue`
